src/ORM/RulesChecker.php

```diff
@@ -46,15 +46,19 @@ class RulesChecker extends BaseRulesChecker
      *
      * ### Options
      *
-     * - `allowMultipleNulls` Allows any field to have multiple null values. Defaults to false.
+     * - `allowMultipleNulls` Allows any field to have multiple null values. Defaults to true.
+     * - `message` Sets a custom error message.
      *
      * @param array<string> $fields The list of fields to check for uniqueness.
      * @param array<string, mixed>|string|null $message The error message to show in case the rule does not pass. Can
      *   also be an array of options. When an array, the 'message' key can be used to provide a message.
      * @return \Cake\Datasource\RuleInvoker
      */
-    public function isUnique(array $fields, array|string|null $message = null): RuleInvoker
-    {
+    public function isUnique(
+        array $fields,
+        array|string|null $message = null,
+        array|string|null $finder = null
```

Comment:
You'll need to add these parameters to the doc strings or linters will complain.